src/taipy/rest/api/resources/cycle.py

```diff
@@ -25,6 +26,15 @@
 REPOSITORY = "cycle"
 
 
+def get_or_raise(cycle_id: str) -> None:
+    """Raise an exception if the data node does not exist."""
```

Comment:
So I don't understand the 'return' statement.

I suspect the function signature and doc are wrong.

Do you really plan to publish this function anyway?